[PL-49306] Update delegate release notes with new expiration policy - breaking change (#6442)
harness/developer-hub: release-notes/delegate.md

<Deleos />

:::danger Breaking change: Updated Delegate Expiration Policy

> Harness has updated the delegate expiration policy from 6 months to 8 months with a 2 month upgrade period.

Author: ```suggestion
Harness has updated the delegate expiration policy from 6 months to 8 months with a 2-month upgrade period.
```

> Harness has updated the delegate expiration policy from 6 months to 8 months with a 2-month upgrade period.

Author: ```suggestion
Harness has updated the delegate expiration policy to 6 months with a 2-month EOL upgrade period.
```

> Six months after a delegate image is released, the delegate reaches End of Support (EOS). Eight months after a delegate image is released, the delegate is End of Life (EOL). Delegates expire if not upgraded 8 months after the image is released.

Author: ```suggestion
Six months after a delegate image is released, the delegate reaches End of Support (EOS). Eight months after a delegate image is released, the delegate is End of Life (EOL). Delegates expire if not upgraded 8 months after the image is released. If delegates are past their EOS date, Harness does not support them. Expired delegates might not work as intended. For issues with expired delegates, Harness Support will request that you upgrade your delegate(s).
```

> Six months after a delegate image is released, the delegate reaches End of Support (EOS). Eight months after a delegate image is released, the delegate is End of Life (EOL). Delegates expire if they are not upgraded 8 months after the image is released.

Author: ```suggestion
Six months after a delegate image is released, the delegate reaches End of Support (EOS). Eight months after a delegate image is released, the delegate is End of Life (EOL). Delegates expire if not upgraded 8 months after the image is released.
```

> Six months after a delegate image is released, the delegate reaches End of Support (EOS). Eight months after a delegate image is released, the delegate is End of Life (EOL). Delegates expire if not upgraded 8 months after the image is released. If delegates are past their EOS date, Harness does not support them. Expired delegates might not work as intended. For issues with expired delegates, Harness Support will request that you upgrade your delegate(s).

Author: ```suggestion
Six months after a delegate image is released, the delegate reaches End of Support (EOS). Eight months after a delegate image is released, the delegate is End of Life (EOL). Delegates expire if not upgraded 6 months after the image is released. If delegates are past their EOS date, Harness does not support them. Expired delegates might not work as intended. For issues with expired delegates, Harness Support will request that you upgrade your delegate(s).
```
